### ISSUE TITLE
Use Anchorage edge shorthand where possible

### ISSUE THREAD

**Issue by ZevEisenberg** [Raizlabs/Eject#23]
`verticalAnchors`, `horizontalAnchors`, and `edgeAnchors` should all be used if possible.

**Comment by KingOfBrian** [comment]
Good idea! I think I may add an analysis pass after the `CodeGenerators` are populated in the `XIBDocument`. This could do the duplicate variable name detection, and coalesce `ConstraintCodeGenerator` to use the edge shorthands.

**Comment by KingOfBrian** [comment]
Should be able to follow a pattern like this to scan statements for 2 or 4 matching generators and replace them with the compound anchors.

https://github.com/Raizlabs/Eject/blob/master/EjectKit/Builder/ActionBuilder.swift#L37

**Comment by ZevEisenberg** [comment]
Now we have `centerAnchors` as well.
